vignettes/iSubGenGuide.Rnw

```diff
@@ -78,23 +78,23 @@ This will allow patient classification based on whether the same relationships a
 \subsection{Applying {\emph{iSubGen}} to an example}
 \label{small-example}
 
-We will demonstrate how to use \emph{iSubGen} with a small example dataset. 
+We will demonstrate how to use \emph{iSubGen} with a small example dataset.
 This example has copy number (CNA) profiles, methylation profiles and coding single nucleotide variant (SNV) data for 30 patients.
```

Comment:
**(style|doc|non-blocking)** Minor point but CNA, SNV acronyms can be introduced above.